src/Skynet.Server/Services/NotificationService.cs

```diff
@@ -55,11 +59,11 @@ async Task process(Session session)
                     session.LastFcmMessage = DateTime.Now;
                     database.Entry(session).Property(s => s.LastFcmMessage).IsModified = true;
                     await database.SaveChangesAsync().ConfigureAwait(false);
-                    Console.WriteLine($"Successfully sent FCM message to {session.FcmToken.Remove(16)} last connected {session.LastConnected}");
+                    logger.LogInformation($"Successfully sent Firebase message to {session.FcmToken.Remove(16)}... last connected {session.LastConnected}");
```

Comment:
I would keep FCM, because Firebase is an entire service collection consisting of storage, crashlytics (unrelated sidenote: which we could use, actually), machine learning and much more.